pool/store/mariadb_store.py

```diff
@@ -0,0 +1,250 @@
+import os
+import yaml
+from pathlib import Path
+from typing import Optional, Set, List, Tuple, Dict
+
+import asyncio
+import aiomysql
+import pymysql
+from blspy import G1Element
+from chia.pools.pool_wallet_info import PoolState
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.coin_solution import CoinSolution
+from chia.util.ints import uint64
+
+from .abstract import AbstractPoolStore
+from ..record import FarmerRecord
+from ..util import RequestMetadata
+
+pymysql.converters.encoders[uint64] = pymysql.converters.escape_int
+pymysql.converters.conversions = pymysql.converters.encoders.copy()
+pymysql.converters.conversions.update(pymysql.converters.decoders)
+
+# We load our configurations from here
+with open(os.getcwd() + "/config.yaml") as f:
+    pool_config: Dict = yaml.safe_load(f)
```

Comment:
Should pass the config as a parameter to the class.

Reply:
adding

Reply:
done